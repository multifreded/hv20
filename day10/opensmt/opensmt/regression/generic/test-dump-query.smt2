(set-option :dump-query true)
(set-option :dump-query-name "dump")
(set-option :output-dir "/tmp/")
(set-option :time-queries true)
(set-logic QF_LRA)
(declare-fun x () Real)
(declare-fun y () Real)
(assert (> x y))
(check-sat)
(exit)
