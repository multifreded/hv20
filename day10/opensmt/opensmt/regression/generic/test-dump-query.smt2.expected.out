; QF_LRA solver query time so far: 0.000000
sat
